### ISSUE TITLE
Problem when generating an fmu: either fails or missing files

### ISSUE THREAD

**Issue by gitting-around** [INTO-CPS-Association/unifmu#12]
There seems to be a problem when creating an fmu from the gui.

When I choose the python backend it throws the following error (same if creating from shell):

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/unifmu/gui.py", line 165, in on_generate
    generate_fmu_from_backend(self.backend_combo.Value, output_path)
  File "/usr/local/lib/python3.8/site-packages/unifmu/generate.py", line 146, in generate_fmu_from_backend
    copytree(src, dst)
  File "/usr/local/Cellar/python@3.8/3.8.5/Frameworks/Python.framework/Versions/3.8/lib/python3.8/shutil.py", line 552, in copytree
    with os.scandir(src) as itr:
FileNotFoundError: [Errno 2] No such file or directory: '/var/folders/z7/s0fzzjg93psb6zr7fqt_wt7xwz41vz/T/tmph4wp3cxf/resources/backends/python_fmu/flatbuffers'
```

When I choose the java backend, it creates the binaries and resources folders with the corresponding files, however no `modelDescription.xml` file. Same if creating from the shell.

I thought I'd try with the csharp backend also, and it creates the binaries and resources folders with no `modelDescription.xml` file. In the resources folder there is only the `launch.toml` file. In this case, if creating from shell, it creates the `modelDescription.xml` file, and the resources folder with only `launch.toml`.

This is running on macos Catalina.

**Comment by clegaard** [comment]
Thanks for submitting the issue.

We are currently in the process of adding additional backends, which requires a bit of refactoring. By mistake, I pushed some unfinished work to the master branch.

I will fix it ASAP :)

**Comment by clegaard** [comment]
solved by f224340

The unfinished backends have been disabled and backends.toml has been updated to include the correct files
